recipe/conda_build_config.yaml

```diff
@@ -337,7 +337,7 @@ fftw:
 flann:
   - 1.9.2
 flatbuffers:
-  - 24.3.25
+  - 24.3.7
```

Comment:
This migration actually regressed the current pinnings, which were already updated by 42c4ce696a20b2de6c62ae925b2606c60e7a1a7c (and feedstocks like tensorflow that had built on top of that). I know multiple migrations are confusing, but please be careful on this.

Reply:
CC @beckermr, IMO the bot should have never opened this PR, as it downgrades the global pinning.

Reply:
Fixed in https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/5b88cc4b7185989988a5fe85090a45d11889a533